spec/tree-view-spec.coffee

```diff
@@ -2588,6 +2588,43 @@ describe "TreeView", ->
           omicronDir = $(treeView.roots[0].entries).find(".directory:contains(omicron):first span")[0]
           expect(omicronDir.title).toEqual("omicron")
 
+      describe "when a file is created within a directory with another squashed directory", ->
+        it "un-squashes the directories", ->
+          jasmine.attachToDOM(workspaceElement)
+          piDir = $(treeView.roots[0].entries).find(".directory:contains(omicron#{path.sep}pi):first")[0]
+          expect(piDir).not.toBeNull()
+          omicronPath = piDir.getPath().replace "/pi", ""
```

Comment:
Since the directories are squashed, the execution of `$(treeView.roots[0].entries).find(".directory:contains(omicron):first")[0].getPath()` returns `<path>/omicron/pi`.
Anyway I have to change `/pi` to `#{path.sep}pi` to make it Windows-friendly.
Commit is on the way! ;)